src/App.tsx

```diff
@@ -1,39 +1,128 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import './App.scss';
 import Carousel from './components/Carousel';
 
-interface State {
-  images: string[];
-}
-
-class App extends React.Component<{}, State> {
-  state = {
-    images: [
-      './img/1.png',
-      './img/2.png',
-      './img/3.png',
-      './img/4.png',
-      './img/5.png',
-      './img/6.png',
-      './img/7.png',
-      './img/8.png',
-      './img/9.png',
-      './img/10.png',
-    ],
-  };
-
-  render() {
-    const { images } = this.state;
-
-    return (
-      <div className="App">
-        {/* eslint-disable-next-line */}
-        <h1>Carousel with {images.length} images</h1>
-
-        <Carousel />
-      </div>
-    );
-  }
-}
+const images = [
+  './img/1.png',
+  './img/2.png',
+  './img/3.png',
+  './img/4.png',
+  './img/5.png',
+  './img/6.png',
+  './img/7.png',
+  './img/8.png',
+  './img/9.png',
+  './img/10.png',
+];
+
+const App: React.FC = () => {
+  const [frameSize, setFrameSize] = useState(3);
+  const [step, setStep] = useState(3);
+  const [itemWidth, setItemWidth] = useState(130);
+  const [animationDuration, setAnimationDuration] = useState(1000);
+
+  return (
+    <div className="App">
+      <h1
+        className="title"
+        data-cy="title"
+      >
+        {`Carousel with ${images.length} images`}
+      </h1>
+
+      <form className="App__form">
+        <label
+          htmlFor="itemWidth"
+          className="App__label"
+        >
+          Item width:
+
+          <input
+            className="App__input"
+            type="number"
+            name="itemWidth"
+            id="itemWidth"
+            min={80}
+            max={200}
+            value={itemWidth}
+            onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+              setItemWidth(+event.target.value);
+            }}
+          />
```

Comment:
make one handler for all fields
```
const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
  const { name, value } = event.target;

  setInputs(prevInputs => ({ ...prevInputs, [name]: +value }))
}
```

```suggestion
            onChange={onChange}
          />
```